perftracker/templates/comparison_id.html

```diff
@@ -155,26 +187,27 @@ <h3 class='pt_collapse expanded' data-toggle='collapse' data-target='#data_{{ g.
     table.ajax.url(link).load(null, true);
 }
 
+var shown_tables = new Set();
```

Comment:
I am really uncomfortable that you describe the state with 3 sets.

shown/hidden can be obtained via
$(element).is(":hidden");   or similar.

I would generate HTML with single Object like 
`
`
```suggestion
var data = {
...
  '1_21': { 
       chart: [....],   // generated by template
       table: null   // <-- this one will be requested later by XHR
       // if needed (may be not, we can determine visibility from the control itself)
       chart_shown: true,
       table_shown: false
    },
   ...
}

```